github-management/kubernetes-repositories.md

```diff
@@ -127,6 +130,62 @@ ecosystem. Carries the endorsement of the Kubernetes community.
      Kubernetes steering committee elections. in the Kubernetes community
    * Repository must be approved by SIG-Architecture
 
+## Creating Repositories
+
+### Non-staging repositories
+
+For non-staging repositories, suggestions on how to create
+a new repository are described below.
+
+  * Ensure that the repo creation request has appropriate approvals
+  as per the rules mentioned above.
+  * Using the organization and repository name mentioned in the repo creation
+  request, create a new repo with default GitHub settings.
+  * Set the description as per the repo creation request.
+  * Clone the newly created repo locally.
+  * Copy the latest contents of [kubernetes-template-project] and create an
+  initial commit with the message *Initial commit from the
+  kubernetes-template-project*.
+  * Make the following changes in the next commit:
+    * If the request references a GitHub team to be listed in the `OWNERS`
+    file, update the `OWNERS_ALIASES` file to remove the steering-committee
+    alias and add a new alias for the team with members populated as per the
+    GitHub team. If the request does not reference a GitHub team, remove the
+    `OWNERS_ALIASES` file.
+    * Update the OWNERS file as per the request. If the repo is a
+    [SIG Repository], add a labels entry for the SIG that the repo belongs to.
+    * Update the `SECURITY_CONTACTS` file as per the request. Note that aliases
+    cannot be used in this case so expand the GitHub team, if specified.
+    * Create a new commit with the message *Update OWNERS, OWNERS_ALIASES and
+    SECURITY_CONTACTS*.
```

Comment:
I ended up updating the README and CONTRIBUTING.md as a part of https://github.com/kubernetes-csi/csi-driver-image-populator/commit/2f85f6a68e637aec1cff3ef3e058faeb378268a5, but I don't think we should make it a hard requirement for members of the GitHub admin team to have context for the repos being created.

I guess it's nice-to-have but eventually it is the repo maintainers' responsibility to update the README and CONTRIBUTING.md files.